README.md

```diff
@@ -229,41 +120,16 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
-
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
 > Give credit to everyone who inspired your codebase.
```

Comment:
- [ ] This is another comments whic has no importance to visitors or viewers, and I think there's no point including it too. In order to fully customize your work, please try removing all templates or comments in this section. It makes your repo professionally documented.